BUILD.md

```diff
@@ -136,14 +136,23 @@ The steps below will set up a Python virtual environment to run Avalon.
    If you are not behind a corporate proxy (the usual case),
    then leave this line commented out.
 
-7. Create a Python virtual environment:
+7. Standalone build does not support communication between listener and enclave manager over the ZMQ.
+   In file `$TCF_HOME/config/singleton_enclave_config.toml set sync_workload_execution flag to 0
```

Comment:
We need to fix listener issue by changing the zmq url to localhost that will fix actual issue with ZMQ.